README_OTHERPLATFORMS.md

```diff
@@ -1,22 +1,20 @@
-# Compiling for RISC-V
-
-How to compile  and run Toit on a SiFive Unmatched dev board or RISC-V VM with QEMU. 
+# Compiling for other platforms (RISC-V, ARM32, ARM64)
 
 | STATUS | |
 | ------------- | ------------- |
-| ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green) | Linux RISC-V environment |
+| ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green) | Linux environment |
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green) | IDF environment |
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green) | IDF compile sources |
-| ![](https://img.shields.io/static/v1?label=&message=FAILURE&color=red) | IDF export > [doesn't compile on RISC-V host](https://github.com/dsobotta/toit-riscv/issues/4) |
+| ![](https://img.shields.io/static/v1?label=&message=FAILURE&color=red) | IDF export > [doesn't compile on RISC-V host](https://github.com/dsobotta/toit/issues/4) |
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green)| Toit generate build files |
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green) | Toit compile sources |
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green)| Toit generate boot snapshot |
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green) | Toit run examples |
-| ![](https://img.shields.io/static/v1?label=&message=PARTIAL&color=yellow) | Cross-compile for RISC-V > [fails to link on Arch](https://github.com/dsobotta/toit-riscv/issues/6)|
-| ![](https://img.shields.io/static/v1?label=&message=TODO&color=orange) | Embedded RISC-V support |
+| ![](https://img.shields.io/static/v1?label=&message=PARTIAL&color=yellow) | Cross-compile > [fails to link on Arch](https://github.com/dsobotta/toit/issues/6)|
+| ![](https://img.shields.io/static/v1?label=&message=TODO&color=orange) | Embedded support |
 
 
-## 1) RISC-V Environment Setup
+## 1) Environment Setup (RISC-V Example)
```

Comment:
Nit: Stick to non-"title case" for all these (for consistency with other titles and the main README)?